README.md

```diff
@@ -28,6 +39,7 @@ set :pushr_default_hooks, -> { true } # stop, start, restart the pushr daemon au
 set :pushr_pid, -> { File.join shared_path, 'tmp', 'pids', 'pushr.pid' }                                    # location of the pushr pid file
```

Comment:
@adam101 Please in a following release change the default `shared_path` values to `current_path` ones.  Let the developer decide if `config` or `tmp` is symlinked into `shared` or not.